### ISSUE TITLE
ag-grid-react failed install with React 19. 

### ISSUE THREAD

**Issue by MorkWood2** [ceolter/ag-grid#9143]
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
``

npm install ag-grid-react  triggered an error due to dependency resolution with React 19.....

I am using NextJS on the latest install.

This is the error I am running into when I npm install ag-grid. 

npm install ag-grid-react

(node:9011) ExperimentalWarning: CommonJS module /opt/homebrew/lib/node_modules/npm/node_modules/debug/src/node.js is loading ES Module /opt/homebrew/lib/node_modules/npm/node_modules/supports-color/index.js using require().
Support for loading ES Module in require() is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: project-demo@0.1.0
npm error Found: react@19.0.0-rc-69d4b800-20241021
npm error node_modules/react
npm error   react@"19.0.0-rc-69d4b800-20241021" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@"^16.3.0 || ^17.0.0 || ^18.0.0" from ag-grid-react@32.3.1
npm error node_modules/ag-grid-react
npm error   ag-grid-react@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.
npm error
npm error
npm error For a full report see:
npm error /Users/markwild/.npm/_logs/2024-10-28T22_12_01_061Z-eresolve-report.txt
npm error A complete log of this run can be found in: /Users/markwild/.npm/_logs/2024-10-28T22_12_01_061Z-debug-0.log

**Comment by kiril-matev** [comment]
Hello @MorkWood2,

Thank you for reporting this.

We've opened a bug for this:
**AG-13297 When running npm install AG Grid with React 19 errors are thrown**

This will be addressed in a patch version by the end of this week.

Thank you for using AG Grid!

**Comment by MorkWood2** [comment]
@kiril-matev Any update on this, it's still not working...

**Comment by kiril-matev** [comment]
@MorkWood2 This issue was addressed in AG Grid v32.3.2 released last week.

Please upgrade to v32.3.2 to resolve this issue.
